### ISSUE TITLE
Moving parameters "numAverages" and "repititionTime"

### ISSUE THREAD

**Issue by AvGladiss** [MagneticParticleImaging/MDF#63]
I suggest to move the parameters "numAverages" and "repititionTime" from /acquisition/drivefield to either /acquisition/receiver or /acquisition.

The number of averages is not part of the drive field but the receiving unit.
As the repitition time depends on the number of averages, it should not be part of the drivefield-group either.

Therefore, I think these parameters fit best into /acquisition/receiver.

**Comment by hofmannmartin** [comment]
For `numAverages` I agree. As for the repetition time... As it is the repetition time of a drive field cycle it should stay in `/acquisition/drivefield`

**Comment by tknopp** [comment]
we are actually facing the issue that we have two repetition times. One of the DF and a second for the focus field in a multi-patch setting. But the later should be a multiple of the former.

**Comment by hofmannmartin** [comment]
The later repetition time should be the number of patches times the drive field repetition time in a quasi static setting, where the focus field can be assumed constant throughout one cycle. Therefore, we could skip this parameter.

**Comment by tknopp** [comment]
yep, indeed. lets keep the number of redundant parameters at a minimum.

**Comment by tknopp** [comment]
numAverages and repetitionTime are receiver properties but they are also linked to the sequence since they define how long the measurement will be. Maybe "/acquisition" is indeed the right category.

**Comment by AvGladiss** [comment]
> As for the repetition time... As it is the repetition time of a drive field cycle it should stay in /acquisition/drivefield
I disagree on this. The repetition time of the drive field is given only by `period`. The drive field itself also does not change when adding a focus field. In a normal case, the drive field period is constant throughout a whole measurement.
The repetition time is a multiple of the the drive field period, but describes the whole sequence, not only the drive field.

**Comment by hofmannmartin** [comment]
You are right.
